pkg/controller/infinispan/resources/config/xsite.go

```diff
@@ -20,51 +20,51 @@ import (
 )
 
 // ComputeXSite compute the xsite struct for cross site function
-func ComputeXSite(infinispan *ispnv1.Infinispan, kubernetes *kube.Kubernetes, service *corev1.Service, logger logr.Logger, xsite *config.XSite) error {
-	if infinispan.HasSites() {
-		siteServiceName := infinispan.GetSiteServiceName()
-		localSiteHost, localSitePort, err := getCrossSiteServiceHostPort(service, kubernetes, logger)
-		if err != nil {
-			logger.Error(err, "error retrieving local x-site service information")
-			return err
-		}
+func ComputeXSite(infinispan *ispnv1.Infinispan, kubernetes *kube.Kubernetes, service *corev1.Service, logger logr.Logger) (error, *config.XSite) {
```

Comment:
Can you update the return type to be `(*config.XSite, error)` as it's more consistent with how we have handled errors elsewhere.

Reply:
Thanks, will fix